fdbbackup/backup.actor.cpp

```diff
@@ -3326,6 +3466,15 @@ int main(int argc, char* argv[]) {
 		DBType dbType = DBType::UNDEFINED;
 
 		std::unique_ptr<CSimpleOpt> args;
+		ParsedArgs parsedArgs;
+
+		const char** newArgV{};
+
+		if (!parsedArgs.reorderArguments(argc, const_cast<const char**>(argv), argc, newArgV)) {
```

Comment:
```suggestion
		if (!parsedArgs.reorderArguments(argc, const_cast<const char**>(argv), newArgC, newArgV)) {
```

Otherwice the code is not understandable.